uefi/src/proto/media/file/mod.rs

```diff
@@ -304,6 +304,17 @@ impl File for FileHandle {
     }
 }
 
+impl Debug for FileHandle {
+    fn fmt(&self, f: &mut Formatter<'_>) -> core::fmt::Result {
+        f.debug_struct("FileHandle")
+            .field("is_regular_file", &self.is_regular_file().unwrap_or(false))
+            .field("is_directory", &self.is_directory().unwrap_or(false))
+            // We can't print the info, as fetching info requires &mut self.
+            .field("info", &"<hidden>")
```

Comment:
Should we instead write something such as `<unavailable>` to more accurately indicate why we can't display this field? I'm just wondering if we won't confuse users (who will be worried/won't know why they can't get a file's `info` when debug printing it, until they read the comment above).

Reply:
I think maybe we should `derive(Debug)` for `FileHandle`. It would just print the internal pointer, but that would be a more transparent representation of what data it contains, which is typical for a `Debug` impl. The fields listed here also aren't real fields of `FileImpl`, so I think it could give the wrong idea to someone trying to understand what a `FileHandle` is by calling `dbg!(...)` or similar.